src/perf_counters.cc

```diff
@@ -47,8 +47,9 @@ size_t PerfCounterValues::Read(const std::vector<int>& leaders) {
       size -= data_bytes;
     } else {
       int err = errno;
-      GetErrorLogInstance() << "Error reading lead " << lead << " errno:" << err
-                            << " " << ::strerror(err) << "\n";
+      GetErrorLogInstance()
+          << "PerfCounterValues: error reading fd:" << lead << " errno:" << err
```

Comment:
this is unrelated, correct? may be best for a separate small CL.

Reply:
Unrelated. It was throwing that at some point and I felt it needed more language.

Reply:
I mean, it is related to another PR approved earlier this week.